pkg/txn/storage/memorystorage/memorytable/batch_utils.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2022 Matrix Origin
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package memorytable
+
+import (
+	"github.com/matrixorigin/matrixone/pkg/container/batch"
+)
+
+type BatchIter func() (tuple []Nullable)
+
+func NewBatchIter(b *batch.Batch) BatchIter {
```

Comment:
Since this is a new beginning, how about we start document all public data structure/function?

Reply:
OK